public/pages/CreateMonitor/containers/CreateMonitor/CreateMonitor.js

```diff
@@ -112,7 +113,6 @@ export default class CreateMonitor extends Component {
         this.props.history.push(`/monitors/${id}`);
       } else {
         console.log('Failed to update:', resp);
-        this.backendErrorHandler('update', resp);
       }
```

Comment:
Why do we not want to add `backendErrorNotification` here?

Reply:
This is good question. 😃
There is another function `updateMonitor()`:
https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/blob/a8350e0f6842fcea59a41754c10a6648df392286/public/pages/MonitorDetails/containers/MonitorDetails.js#L140
For the button "Disable/Enable" in the monitor details page, that's the only function being called after clicking it, so I added `backendErrorNotification()` there.
However, all the other actions of updating the monitor/trigger eventually call the function `updateMonitor()` above, so I have to remove the code of displaying toast notification in the other functions that call `updateMonitor()` to avoid duplicate notifications.

Reply:
Ah ok, that makes sense. Thanks!

Reply:
No problem. 😁